### PR TITLE
New seed and amenity

### PR DESCRIPTION
**Pull request by JMD1986**
someone check this out

### CODE REVIEW

**Comment by Graciexia** [comment]
:+1:

**Comment by alejandroereyes** [comment]
:+1:

**Comment by BenHand** [comment]
:+1: :100:

File: app/models/amenity.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Amenity < ActiveRecord::Base
+  belongs_to :room
```
Comment (mlgiardina) [inline app/models/amenity.rb]: shouldn't amenities `have_many` rooms?

Comment (jah2488) [inline app/models/amenity.rb]: Yah I'm pretty sure `Amenity` is going to `have_many` rooms and `Room` is going to `have_many` amenities.

Gonna need a join table.

Comment (JMD1986) [inline app/models/amenity.rb]: do you guys mean "has_many"?

Comment (mlgiardina) [inline app/models/amenity.rb]: yes

Sincerely,

Matthew Giardina

**Comment by jah2488** [comment]
:+1:

**Pull request merged**